Composer/packages/client/src/components/GetStarted/GetStarted.tsx

```diff
@@ -46,129 +45,69 @@ export const GetStarted: React.FC<GetStartedProps> = (props) => {
-      />
-      <Stack horizontal>
-        <Stack.Item grow={0} styles={{ root: { width: 250, borderRight: '1px solid #3d3d3d', padding: 38 } }}>
-          <h2 style={h2Style}>{botProject?.name}</h2>
+    <Panel headerText={botProject?.name} isOpen={props.isOpen} onDismiss={props.onDismiss}>
+      <Stack>
+        <Stack.Item grow={0}>
           <p>
-            {formatMessage('File Location:')} <span style={{ textOverflow: 'ellipses', fontSize: 12 }}>{location}</span>
+            {formatMessage('File Location:')}
+            <span
+              style={{
+                display: 'inline-block',
+                overflowWrap: 'break-word',
+                maxWidth: '100%',
+                fontSize: 12,
+              }}
+            >
+              {location}
+            </span>
           </p>
         </Stack.Item>
-        <Stack.Item styles={{ root: { padding: 38 } }}>
+        <Stack.Item>
           <h3 style={h3Style}>{formatMessage('Next steps')}</h3>
-          <Stack horizontal verticalFill tokens={{ childrenGap: 40 }}>
-            <Stack.Item>
-              {formatMessage('Customize')}
-              <ul style={ulStyle}>
-                <li style={liStyle}>
-                  <Link href={linkToPackageManager} styles={linkStyle} onClick={linkClick}>
-                    {formatMessage('Add and remove packages')}
-                  </Link>
-                </li>
-                <li style={liStyle}>
-                  <Link href={linkToLGEditor} styles={linkStyle} onClick={linkClick}>
-                    {formatMessage('Edit what your bot says')}
-                  </Link>
-                </li>
-                <li style={liStyle}>
-                  <Link href={linkToLUEditor} styles={linkStyle} onClick={linkClick}>
-                    {formatMessage('Train your language model')}
-                  </Link>
-                </li>
-                <li style={liStyle}>
-                  <Link href={linkToConnections} styles={linkStyle} onClick={linkClick}>
-                    {formatMessage('Connect your bot to new services')}
-                  </Link>
-                </li>
-              </ul>
-            </Stack.Item>
-            <Stack.Item>
-              {formatMessage('Publish')}
-              <ul style={ulStyle}>
-                <li style={liStyle}>
-                  <Link href={linkToProvision} styles={linkStyle} onClick={linkClick}>
-                    {formatMessage('Create a cloud hosting environment')}
-                  </Link>
-                </li>
-                <li style={liStyle}>
-                  <Link href={linkToPublish} styles={linkStyle} onClick={linkClick}>
-                    {formatMessage('Publish updates to the cloud')}
-                  </Link>
-                </li>
-              </ul>
-            </Stack.Item>
-          </Stack>
+          {formatMessage('Customize')}
+          <ul style={ulStyle}>
```

Comment:
nit: I suggest putting all your urls and their texts in an array:
```
const data = React.useMemo(() => [{url: ..., text: formatMessage(...)}, ...], []);
...
...
data.map(item => <li style={liStyle}><Link href={item.url} ...>{item.text}</Link></li>
```
so you dont repeat yourself.

Reply:
The form and format of these are rapidly changign right now. I'll come back and optimize the handling of this content later.